password_security/models/res_users.py

```diff
@@ -156,3 +156,20 @@ def _set_encrypted_password(self, encrypted):
                 'password_crypt': encrypted,
             })],
         })
+
+    # Dirty fix because this was failing when any theme was installed
+    # I think this should be a fix in the core.
+
+    @api.multi
+    def _is_system(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_system()
+
+    @api.multi
+    def _is_superuser(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_superuser()
+
+    ############ End of dirty fix ###############
```

Comment:
got it, and what do you think about https://github.com/Vauxoo/odoo/pull/189, in order to avoid the cherry-pick made in this pr